ab_tool/models.py

```diff
@@ -76,6 +76,7 @@ def to_json(self):
             "notes": self.notes,
             "uniformRandom": bool(self.assignment_method == self.UNIFORM_RANDOM),
             "tracks": [{"id": t.id, "weighting": t.get_weighting(), "name": t.name,
+                        "originalName": t.name, "newName" : t.name, "editing": False,
```

Comment:
Could we simplify this a bit and just use two name keys: `name` and `originalName`?  I think it would be easier to follow and I'll comment below where I think it could reduce some code.